nidm_results_display.m

```diff
@@ -25,7 +25,15 @@
         addpath(fullfile(fileparts(mfilename('fullpath')), 'lib'));
     end
     
+    graph = jsondoc.('x_graph');
+    % Deal with sub-graphs (bundle)
```

Comment:
Could you perhaps add a comment just explaining a bit more why there would be a subgraph?